oas_docs/output/kibana.serverless.staging.yaml

```diff
@@ -33502,33 +33502,42 @@ tags:
     x-displayName: Saved objects
   - description: Manage and interact with Security Assistant resources.
     name: Security AI Assistant API
+    x-displayName: Security AI assistant
   - description: >-
       You can create rules that automatically turn events and external alerts
       sent to Elastic Security into detection alerts. These alerts are displayed
       on the Detections page.
     name: Security Detections API
+    x-displayName: Security detections
   - description: >-
       Endpoint Exceptions API allows you to manage detection rule endpoint
       exceptions to prevent a rule from generating an alert from incoming events
       even when the rule's other criteria are met.
     name: Security Endpoint Exceptions API
+    x-displayName: Security endpoint exceptions
   - description: Interact with and manage endpoints running the Elastic Defend integration.
     name: Security Endpoint Management API
+    x-displayName: Security endpoint management
   - description: ''
     name: Security Entity Analytics API
+    x-displayName: Security entity analytics
   - description: >-
       Exceptions API allows you to manage detection rule exceptions to prevent a
       rule from generating an alert from incoming events even when the rule's
       other criteria are met.
     name: Security Exceptions API
+    x-displayName: Security exceptions
   - description: Lists API allows you to manage lists of keywords, IPs or IP ranges items.
     name: Security Lists API
+    x-displayName: Security lists
```

Comment:
@approksiu @nastasha-solomon do we refer to them as `Security value lists` or just `Security lists`?

Reply:
In the UI docs, we use `value lists`: https://www.elastic.co/guide/en/security/current/value-lists-exceptions.html

In the API docs, we use `lists`: https://www.elastic.co/guide/en/security/current/lists-index-api-overview.html

Also, I'm not sure whether you need to specify that these lists are for Security only. The distinction might be helpful if you can set up exceptions that use lists for Kibana and Observability rules. Otherwise, `value lists` or `lists` should suffice.